### DISCUSSION TITLE
QuickStart stuck in "Retrying in 10 seconds..." loop.

### DISCUSSION THREAD

**Discussion by bredd** [wgu-opensource/osmt#165] [Asking for Help] [answered]
I "git cloned" the latest from the "develop" branch. Configured my Okta instance, ran "./osmt_cli.sh -i", updated my osmt-quickstart.env file with the Okta info, and then ran "./osmt_cli.sh -q".

It takes quite a while to download and build the container images. once there it goes into a loop with "INFO: Retrying in 10 seconds." The last error reported before the retry is "java.net.UnknownHostException: osmt_quickstart_elasticsearch_1".

I cleared everything and started from scratch with the steps listed above and got the same result.

When I check the Docker UI it shows "osmt_quickstart-elasticsearch-1" running on port 9200. 

I'm running Docker on Windows WSL2. I run "osmt_cli.sh" using Git Bash. I don't think either of those is the problem.

Any suggestions?

**Comment by JohnKallies** [comment, marked as answer]
I had the same experience in Windows (see my reply to your earlier post).

With your stack still running, run `docker ps` and compare your container names with the URLs in your osmt-quickstart.env file. Your Spring logout shows underscores, but your Docker container name has hyphens. Update your env file to match the existing container names.

See https://github.com/wgu-opensource/osmt/discussions/146 for more about this...

**Reply by bredd** [reply]
Thank you! I also saw your other answer:

I changed ELASTICSEARCH_URI to "http://osmt_quickstart-elasticsearch-1:9200"
(It was previously http://osmt_quickstart_elasticsearch_1:9200)

I changed REDIS_URI to "osmt_quickstart-redis-1:6379"
(It was previously "osmt_quickstart_redis_1:6379

That solved most of the problem but it still can't connect to the database. I haven't found where to change the database URI. I expect that's my lingering issue.

I wonder why it's different on Windows from the other platforms?
